### PR TITLE
chore: Add targeted Flag to example config

### PR DESCRIPTION
**Pull request by agardnerIT**
Adding a new flag because in https://github.com/open-feature/killercoda/pull/5 I'm trying to explain targeting.

This example is explicit in that:
- Everyone gets `first` except
- Those with `email` containing `@openfeature.dev` - get `second`
- Those with `userAgent` containing `Chrome` - get `third`.

### CODE REVIEW

File: config/samples/example_flags.flagd.json [hunk 1]
```diff
@@ -122,6 +122,30 @@
           }, null
         ]
       }
+    },
+    "targetedFlag": {
+      "returnType": "string",
```
Comment (beeme1mr) [inline config/samples/example_flags.flagd.json:127]: This property isn't used.

Comment (beeme1mr) [inline config/samples/example_flags.flagd.json:127]:
```suggestion
```

The `returnType` property isn't used.

Comment (agardnerIT) [reply config/samples/example_flags.flagd.json]: Removed

**Comment by agardnerIT** [comment]
I've also added (what I think) is the correct YAML definition. Please do check that closely.

File: config/samples/example_flags.flagd.yaml [hunk 2]
```diff
@@ -80,9 +80,29 @@ flags:
...
+        - var:
+          - "email"
+        - "@openfeature.dev"
+      - 'second'
+      - null
+        - var:
+          - "userAgent"
+        - "Chrome"
+      - 'third'
+      - null
```
Comment (beeme1mr) [inline config/samples/example_flags.flagd.yaml:102]:
```suggestion
if:
- in:
  - "@openfeature.dev"
  - var: email
- second
- in:
  - Chrome
  - var: userAgent
- third
-
```

**Review by beeme1mr — COMMENTED** [review]
The YAML looked wrong. I used the JSON to YAML converter to generate the suggested changes.

**Review by beeme1mr — APPROVED** [review]

**Review by toddbaert — APPROVED** [review]

**Pull request merged**